src/gui/UpdateProgressTracker.h

```diff
@@ -0,0 +1,15 @@
+#ifndef _UPDATE_PROGRESS_TRACKER_H
```

Comment:
We aren't entirely consistent on this, but as we add new, or as I edit headers, I've been pushing toward including the path in the include guard. This makes accidental duplication less likely from some define elsewhere or in another lib.

so \_SRC_GUI_UPDATE_PROGRESS_TRACKER_H\_